DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/AllDelegates/_SearchableDelegateCard.cshtml

```diff
@@ -0,0 +1,71 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common

```

Comment:
There's going to be a fair bit of refactoring in this file once it comes to implementing the search/filter stuff but looks OK for now